code/modules/events/_event.dm

```diff
@@ -30,8 +30,9 @@
 	var/triggering	//admin cancellation
 	var/auto_add = TRUE				//Auto add to the event pool, if not you have to do it yourself!
 
-	/// Whether or not dynamic should hijack this event
-	var/dynamic_should_hijack = FALSE
+	
+	var/dynamic_should_hijack = FALSE	// Whether or not dynamic should hijack this event	
+	var/can_spawn_after_shuttlecall = FALSE	// Prevents the event from spawning after the shuttle was called
```

Comment:
Why is the name of the variable the literal opposite of what it does. This doesn't allow them to spawn after shuttlecall, this prevents them from spawning after shuttle call if true.